app/interactors/projects/create/prepare_params.rb

```diff
@@ -0,0 +1,13 @@
+module Projects
+  class Create
+    class PrepareParams
+      include Interactor
+
+      delegate :project, to: :context
+
+      def call
+        ProjectMailer.project_created(project, user).deliver_later
```

Comment:
Incorrect interactor name. You should prepare project params here or name interactor "SendNotification"